contrib/completion/zsh/_docker

```diff
@@ -1494,8 +1494,10 @@ __docker_node_subcommand() {
                 $opts_help \
                 "($help -a --all)"{-a,--all}"[Display all instances]" \
                 "($help)*"{-f=,--filter=}"[Provide filter values]:filter:->filter-options" \
+                "($help)--format=[Format the output using the given go template]:template: " \
```

Comment:
yes, just a little convention in the script when an option needs a argument but we don't provide values.